jupyterhub/config/jupyterhub_config.py

```diff
@@ -133,6 +135,13 @@ async def get(self):
         return self.redirect(self.get_next_url(user))
 
 
+class AppAuthenticatorLogoutHandler(LogoutHandler):
```

Comment:
We should just add 1-2 sentences of explanation here : why do we do this? And Add a `  #TODO: use API when available https://github.com/jupyterhub/jupyterhub/issues/3688`)